src/cards/enfeeble.ts

```diff
@@ -0,0 +1,64 @@
+import * as Unit from '../entity/Unit';
+import { CardCategory } from '../types/commonTypes';
+import type Underworld from '../Underworld';
+import { playDefaultSpellAnimation, playDefaultSpellSFX } from './cardUtils';
+import { Spell } from './index';
+import { CardRarity, probabilityMap } from '../types/commonTypes';
+import { getOrInitModifier } from './util';
+
+const enfeebleId = 'Enfeeble';
+const statChange = -5;
+const spell: Spell = {
+  card: {
+    id: enfeebleId,
+    category: CardCategory.Curses,
+    sfx: 'enfeeble',
+    supportQuantity: true,
+    manaCost: 20,
+    healthCost: 0,
+    expenseScaling: 1,
+    probability: probabilityMap[CardRarity.UNCOMMON],
+    thumbnail: 'spellIconEnfeeble.png',
+    animationPath: 'spell-effects/spellEnfeeble',
+    description: ['spell_enfeeble', (statChange).toString()],
+    effect: async (state, card, quantity, underworld, prediction) => {
+      // .filter: only target living units
+      const targets = state.targetedUnits.filter(u => u.alive);
+      if (targets.length) {
+        playDefaultSpellSFX(card, prediction);
+        //await playDefaultSpellAnimation(card, targets, prediction);
+        for (let unit of targets) {
+          Unit.addModifier(unit, enfeebleId, underworld, prediction, quantity);
+        }
+      }
+      return state;
+    },
+  },
+  modifiers: {
+    add,
+    remove,
+  },
+};
+
+function add(unit: Unit.IUnit, underworld: Underworld, prediction: boolean, quantity: number = 1) {
+  const modifier = getOrInitModifier(unit, enfeebleId, { isCurse: true, quantity, }, () => {
+    //no first time setup
+  });
+
+  unit.damage += statChange * quantity;
+
+  // Ensure damage doesn't go below 0
+  unit.damage = Math.max(unit.damage, 0);
```

Comment:
It'd be funny if making a unit deal negative damage meant they healed their target.  I know this doesn't make any sense, but let's try it anyway to pleasantly surprise players who try it

Reply:
We could even add the heal animation when damage is dealt that it negative outside of the heal spell to show that it's on purpose.

Reply:
> We could even add the heal animation when damage is dealt that it negative

Just to clarify: Do you want the heal animation built into Unit.takeDamage()?

Reply:
I think so... If damage dealt is ever negative we __always__ play the heal animation right?
I'm thinking of potions... all heal spells... I think that's it. 
Is there ever any reason why we wouldn't want this?
Maybe it should go in a modifier instead? I'm open to ideas

Reply:
Nope, I love the idea. Just double checking you meant in takeDamage and not in the modifier or something 😁